files/www/cgi-bin/mesh

```diff
@@ -90,6 +90,7 @@ $tactical = nvram_get("tactical");
 $config = nvram_get("config");
 $config = "not set" if $config eq "" or not -d "/etc/config.mesh";
 ($my_ip) = get_ip4_network(get_interface("wifi"));
+$node_desc = `/sbin/uci get system.\@system[0].description`;
```

Comment:
this will throw errors in the unit test.  change from the `cmd` syntax to $(cmd) syntax